examples/aws-databricks-uc/unity_catalog_infra.tf

```diff
@@ -34,6 +34,19 @@ data "aws_iam_policy_document" "passrole_for_uc" {
       values   = [var.databricks_account_id]
     }
   }
+  statement {
+    effect  = "Allow"
```

Comment:
Should we add `sid = "ExplicitSelfRoleAssumption"`? The template at `modules/aws-databricks-unity-catalog/uc_cross_account_role.tf` has that.

Reply:
Done